assistedInstaller.py

```diff
@@ -65,7 +65,6 @@ def download_iso_with_retry(self, infra_env: str) -> None:
         else:
             logger.error(f"Failed to download the ISO after {retries} attempts")
 
-
```

Comment:
`rebase -i` to split this commit, reorder, and squash it with the commit that addressed the code above.